translations.yml

```diff
@@ -1090,18 +1090,23 @@
 
 - en: 'Editors:'   # gender-neutral, plural. Precedes list of more than one person who edited the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Edition'.
   fr: 'Rédaction :'
+  de: 'Redaktion:'
 
 - en: 'Update Editor:'   # gender-neutral, singular. Precedes list of one person who updates the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Update Edition'.
   fr: 'Rédaction des mises à jour:'
+  de: 'Redaktion für Aktualisierungen'
```

Comment:
This reads strange. (It already seems a bit too much for the first translation.)

If this follows the first term (“Redaktion”), “Aktualisierungen” or “Zuständig für Aktualisierungen” might work better.

Reply:
Not sure, if I understand that right. You mean only "Aktualisierungen" without "Redaktion" or anything of that sort? I can see how "Redaktion für Aktualisierungen" reads strangely in german, but wouldn't something be missing from the english "Update Editor", if we simply omitted it?

Reply:
Yes, that’s what I mean, but let’s by all means challenge it.

Quick first thought: I find “update editor” a bit strange already, I must say.

Then, where can we see this in action? (Did a quick search but couldn’t find anything. Spread superthin, so appreciate a pointer to review.)

Reply:
See the footer here: https://www.w3.org/WAI/tutorials/images/decision-tree/

> Update Editor: Brian Elton.

Reply:
Thanks, appreciated! Yes—my view is that “Aktualisierungen” would work well here. (Test it against “Updates”, which I think would work in English, too?)